### ISSUE TITLE
Infodump (pinned messages)

### ISSUE THREAD

**Issue by Baudshaw-CommunityManager** [kurzgesagt-life/suggestions#4]
Player: A player who is built with clothing to the player liking, so the player's appearance is dependent on the players liking
Bird companion: basically in my idea the player first starts a random bird and gets more accessories to put on the bird

What: starts at the starting of the civilization, growing up in the Scale

When: progressing from the start of the civilization to class omega(then the game will be considered beaten)

Place: a random place around the world with its unique resources, people, etc.(that's the researcher's job)

Other: maybe PvE, and high scores to speedrun the thing
inspired by kurzgesagt videos
random natural events
I feel like the game is way too big for us few devs, so we need a small version of it first on which we can expand first
we can start by random resource levels and see where we can go from there
SwarthToday at 3:05 PM
if we can implement some type of procedural generation, the game could have replayability
McVeteran (Head Researcher)Today at 10:11 AM
Possible Game Achievements 

Existence! - Start the game for the first time

Heating Up – Use fire to keep warm at night

We Stay Here Now – Construct enough structures for your species to stay in

Food Security At Last – Agriculture takes shape

Palokar – Develop a written form of language

It’s A City! – 100000 population

Spinning Lakes - Build a water wheel to gain energy 

Don’t Worry You’ll Heal – Herbology is developed

Life There? – Biology is developed

And That’s An Atom – Chemistry is developed

Gravity! – Physics is developed

It Goes Puff – The steam engine is invented

May The Energy Be With Me – Generate 60 joules of energy per second

WWW – The internet is invented

I Split The Atom – Create a nuclear bomb

Delicious Power - Build a nuclear power plant

Almost There! – Being able to use all the energy available on Earth

It’s Only The Beginning – Type I Civilisation

-SPACE BRANCH-

Take to the Skies - Form a space agency

Round and Round We Go - Send your first satellite into orbit

Earth Looks Cool From Up Here - Send the first human/duck into orbit

A Flying Lab - Launch a space station

One Small Step For Man… – Arrive on the moon

One Giant Leap For Mankind – Set the foundation for a lunar colony

The Seed of The Future - Build a lunar base

-END OF SPACE BRANCH-
McVeteran (Head Researcher)Today at 8:52 AM
I suddenly have an idea:
The moment you reach Kardashev Type III, the screen goes black, shows the credits, and reads, "You have come far beyond what any civilization on Earth has done, and I'm proud of you. Who am I? Well...this is me."
The black screen slowly fades and reveals the god from The Egg, and animation cues, with the god explaining to the player and stuff. He concludes with "And now I will send you off to find your origins. How you came to be. I wish you luck, [Username]. See you in a bit."
You watch as the civilization you built reverse back in time, to your first village, and you watch even the first Homo sapiens disappear. 

"East Africa, Earth. Four million years ago. Australopithecus afarensis."
What if at the end of the game you meet "god" from the egg
Castle Bravo(Researcher)08/29/2020
So I have an idea for variables that involve the individuals of the species. 
Durability
Strength
Intelligence
Durability determines how well individuals can survive in certain environments. Each biome has a different durability rating. If you have more than the required durability rating, you will survive 100% of the time in that biome unless a disaster happens. 

Strength determines how quickly you can access and obtain certain raw materials or resources, and also determines how well your species can survive certain disasters(but not all). It also helps you to make technology faster(this being another element that I can get into)

Intelligence determines how quickly you can unlock the building of technologies, as well as prevents social collapse, and helps advance RNG for the "accidental" discovery of technology or something else. (you cannot find future technologies without a certain "intelligence" value.)

All three of these variables are built upon by Evolution Points, which are granted depending on whether you have certain species variables, that determine the health of your species, above a certain level, then a timer starts, and you gain Evolution Points afterward
Beaverboithe1st08/28/2020

@Toby56 Intelligence is the way you apply knowledge and research, so, if you don't have the intelligence bar, the player won't know what is happening, also, how would we measure knowledge and research? research wouldn't be a bar, it would be more like stats on the thing that you are looking, like a planet and a city, or a country, or alien. but, on the other hand, even if you didn't have the knowledge that you would have, you can use intelligence and apply it to come up and discover the knowledge available. also, it didn't stop millions of years ago. our intelligence is far greater than the intelligence of people in the 50's. I agree with the second statement, though, our advancements in the kardashev scale should be from achievements and advances in society. but, you should be tested beforehand so that we can test the player and see if they do something useful with such achievement or advancement. the third statement is partial, on one hand, yes, you have a point, it can be used to build facilities and send expeditions, but without the bar, the player won't ever really know how intelligent a species is. the knowledge and research are crucial in the advancements in the game, yes, but so is intelligence. the fourth sentence is correct. that is a GREAT idea. we will add that to the game. the fifth one is also correct. one reference is not enough to make a game. but, it is very detailed in the way we want to make the game. please use more references from different videos or other people or your drawings, that will expand your creative thinking and abstract way of making the game! also, if you read this far, you're a legend, and I hope you stay in our team!

Intelligence unlocks the ability to get stuff and know it exists plus craft it
Energy points: Making structures (City's, Rockets, etc)

Intelligence points: unlocking the availability to make different 
structures

I'd say that one energy point should be one joule and you need 1000 points to move up the first 0.1 in the Kardashev Scale and the amount of points you need gradually increases as you get further along the Scale

I still think that before we start implementing this stuff we should complete my rectangular mockup prototype. All it'll be is rectangles walking around and discovering one or two things to get an idea of what the game will play like

Over time, through your actions in the game, you earn "evolution points" or something. You can then use those points and put them to work evolving your species. The smarter they get, the more things they discover, and thus the more evolution points the player gets. Every minute (seems like an appropriate amount of time) that the player is online an in-game year passes. Every decade you get an "event" through which you can change the course of your species' evolution and advancement.
O
so gameplay will consist of guiding humanity through the kardeshev scale. what if we got to decide what path humanity takes to reach there? like global surveillance state vs fluffy friendly utopia
we should (if they say yes) keep them updated on the game
I'll announce what we have atm
but we're making it unofficial

@squak, the game designer no we're not. We're making an unofficial version using rectangles as placeholders, then we'll present it to Kurzgesagt and see if we can make it an official game.

@Kip [C# & Unity] Head Programmer the birds can be companions
Beaverboithe1st08/25/2020

maybe the intelligence creatures can be aliens?

DanB69 ( Researcher)08/25/2020
I thought maybe with the start of humanity and with a sort of intro at the beginning that shows a quick video of evolution from single cellular organisms to the first humans. then the game starts properly.

So, just to get something straight: is this gonna be a single-player Rpg-style (not an Rpg, but that sorta style, with movement controls and a player Sprite), or is it gonna be a single-player omniscient-style game? Omniscient-style would be like you control the civilization itself and have views of the planet(s) that you use rather than a single character?

so our game is going to be a single-player 2d with a progressive system to get higher up the kardashev scale

**Comment by sudo-carson** [comment]
this info is already on the github project

https://github.com/orgs/kurzgesagt-life/projects/1

**Comment by dummiedummie** [comment]
Baudshaw can you credit the people who made the ideas?

**Comment by GKBG** [comment]
Not now pls

На чт, 3 сеп 2020 г., 0:39 dummiedummie <notifications@github.com> написа:
